test/sqlite.test.js

```diff
@@ -24,11 +25,18 @@ describe('sqlite3 connector', function() {
       loc: 'GeoPoint',
       approved: Boolean
     });
+
+    ModelWithPKString = db.define('ModelWithPKString', {
+      id: {type: String, id: true},
+      content: {type: String},
+    });
   });
 
   it('should run migration', function(done) {
     db.automigrate('PostWithBoolean', function() {
-      done();
+      db.automigrate('ModelWithPKString', function() {
```

Comment:
I think you can call `db.automigrate(['PostWithBoolean', 'ModelWithPKString'])`

Reply:
2 years later, yeah sure I can :-P

Reply:
> I just stumbled upon this PR, may I say something about it?
> In Sqlite3, only a column with the type "INTEGER PRIMARY KEY" is an alias for the RowID. Therefore, 'getInsertedId' should only return the lastID (=RowID) if the primary key consists of exactly one INTEGER column.
> Instead of querying for 'NOT-TEXT', you may therefore want to ask for "INTEGER and NOT composite-ID"

+1, I think `_isPrimaryKeyInteger` is better.